tools/doc/html.js

```diff
@@ -8,6 +8,15 @@ var typeParser = require('./type-parser.js');
 
 module.exports = toHTML;
 
+// customized heading without id attribute
+var renderer = new marked.Renderer();
```

Comment:
Wouldn't really fit the style of this file. I guess we shoul to es6ify the doctool later.